omod/src/main/resources/apps/ugandaemr_reports_app.json

```diff
@@ -39,6 +59,14 @@
         "url": "ugandaemrreports/reports.page",
         "icon": "icon-book",
         "order": 110
+      },{
+        "id": "ugandaemrreports2019.homepageLink",
+        "extensionPointId": "org.openmrs.referenceapplication.homepageLink",
+        "type": "link",
+        "label": "UgandaEMR Reports 2019",
```

Comment:
@dbaluku 

1. Rename to UgandaEMR Reports - 2019 HMIS Tools 

2. The link also appears after database backup so find a way to force it to be just after to the UgandaEMR Reports link. I think we should rename it to UgandaEMR Reports - 2014 HMIS Tools

Reply:
Done